### DISCUSSION TITLE
Extends the object initializer syntax to the construction completion object

### DISCUSSION THREAD

**Discussion by zms9110750** [dotnet/csharplang#5985] [Language Ideas]
Now, I usually use tuple assignment to write single line code.
When I encounter an object that needs to be obtained using a method, I either declare a local variable or call the method multiple times.
Can the object initialization syntax be extended to the constructed object?
This is more eye-catching when assigning values to the same object.

now:
```
slider.max=10;
slider.value =5;
slider.min=2;
```
extends:
```
slider{ max=10 ,value =5 , min=2 };
```

**Comment by svick** [comment]
Similar suggestion: https://github.com/dotnet/csharplang/issues/803.
